enumerable.rb

```diff
@@ -0,0 +1,102 @@
+# Custom Enumerable Methods
+module ::Enumerable
+  # Calls block with the item for each item in enum.
+  def my_each
+    return to_enum unless block_given?
+
+    length.times { |n| yield self[n] }
+    self
+  end
+
+  # Loops, Calling The Block With Item And Index
+  def my_each_with_index(&block)
+    for index in 0...count
+      block.call(self[index], index)
+    end
+  end
+
+  # Returns an array containing all elements of enum for which the given block returns a true value.
+  def my_select
+    return to_enum unless block_given?
+
+    array = []
+    my_each do |n|
+      array << n if yield n
+    end
+    array
+  end
+
+  # Passes each element of the collection to the given block.
+  # The method returns true if the block never returns false or nil.
+  def my_all?(&block)
+    my_each do |item|
+      return false if block.call(item) != true
+    end
+
+    true
+  end
+
+  # Passes each element of the collection to the given block.
+  # The method returns true if the block ever returns a value other than false or nil.
+  def my_any?(&block)
+    my_each do |item|
+      return true if block.call(item) == true
+    end
+
+    false
+  end
```

Comment:
### #my_all?
This method is having the same miss behavior as the #my_all method. please refer to this [doc](https://apidock.com/ruby/v2_5_5/Enumerable/any%3F).

### [__Optional__]
Please test all of your code by comparing it with the #any? method before submitting for a new review.

Reply:
Fixed in
https://github.com/epinczinger/ruby_enumerables/pull/7/commits/76c8f868342e3edf70ca3969966ffe792dc9235c